src/openforms/js/components/admin/form_design/registrations/objectsapi/SelectboxesObjectsApiVariableConfigurationEditor.js

```diff
@@ -0,0 +1,132 @@
+import {useFormikContext} from 'formik';
+import isEqual from 'lodash/isEqual';
+import React, {useContext} from 'react';
+import {FormattedMessage} from 'react-intl';
+import {useAsync, useToggle} from 'react-use';
+
+import {APIContext} from 'components/admin/form_design/Context';
+import Field from 'components/admin/forms/Field';
+import FormRow from 'components/admin/forms/FormRow';
+import {Checkbox} from 'components/admin/forms/Inputs';
+import ErrorMessage from 'components/errors/ErrorMessage';
+
+import {MappedVariableTargetPathSelect} from './GenericObjectsApiVariableConfigurationEditor';
+import {asJsonSchema} from './utils';
+import {fetchTargetPaths} from './utils';
+
+export const SelectboxesEditor = ({
+  variable,
+  components,
+  namePrefix,
+  index,
+  mappedVariable,
+  objecttype,
+  objectsApiGroup,
+  objecttypeVersion,
+  backendOptions,
+}) => {
+  const {csrftoken} = useContext(APIContext);
+  const [jsonSchemaVisible, toggleJsonSchemaVisible] = useToggle(false);
+  const {setFieldValue} = useFormikContext();
+  const {transformToList = []} = backendOptions;
+
+  const transformationNeeded = transformToList?.includes(variable.key);
```

Comment:
```suggestion
  const transformationNeeded = transformToList.includes(variable.key);
```
the nullish-coalesce `?.` is not necessary anymore because we default `tranformToList` to an empty array on L33

Reply:
And this is exactly what we discussed yesterday about the feedback I receive..apologies, I will update it